.github/fabricbot.json

```
@@ -3532,7 +3532,7 @@
           {
             "name": "addReply",
             "parameters": {
-              "comment": "Looks like this PR hasn't been active for some time and the codebase could have been changed in the meantime.\nTo make sure no breaking changes are introduced, please remove the `pending ci rerun` label to kick off a new CI run."
+              "comment": "Looks like this PR hasn't been active for some time and the codebase could have been changed in the meantime.\nTo make sure no breaking changes are introduced, please leave an `/azp run` comment here to rerun the CI pipeline again and confirm success before merging the change."
```

Nit: re-run again is redundant.

---

@amcasey do you mean e.g.
```suggestion
              "comment": "Looks like this PR hasn't been active for some time and the codebase could have been changed in the meantime.\nTo make sure no breaking changes are introduced, please leave an `/azp run` comment here to rerun the CI pipeline and confirm success before merging the change."
```